_includes/header.html

```diff
@@ -1,11 +1,9 @@
 <header class="navigation" role="banner">
   <section class="itemgroup">
-    <a href="/" class="logo" title="RustFest">
+    <a href="/" class="logo" title="RustFest" alt="RustFest">
       {% include icons/logo-header.svg %}
-      <span class="site-title">
-        RustFest
-      </span>
     </a>
+    <span style="color:white;">{{ site.location.city }}, {{ site.location.country }} - {{ site.dates.short_form }}</span>
```

Comment:
I don't like how the `ul` at line 8 pushes this `span` on two lines. Can any CSS wizard help out here?

Reply:
without testing, have you checked if the `section.itemgroup` has a `max-height`?
Another one might be `clear: both;` if there was a `float` somewhere

Reply:
Also `style=...` should go into `_sass/components/_navigation.scss` around line 11.